PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeTests.cs

```diff
@@ -1,12 +1,13 @@
-﻿using UnityEngine;
+﻿#if !UNITY_WSA && !UNITY_WSA_10_0
```

Comment:
Any reason why code is getting skipped for UNITY_WSA AND UNITY_WSA_10_0 ?

Reply:
Please review the known limitations [here](https://docs.unity3d.com/Packages/com.unity.test-framework@1.0/manual/index.html)

Reply:
@crimsonred  Were you able to run any example test on UWP to verify that it is working as expected.

Reply:
yes, the example is running on UWP.